File: notification-service/src/main/java/notification/service/dto/NotificationRequest.java
package notification.service.dto;


import com.fasterxml.jackson.annotation.JsonProperty;
import notification.service.enums.NotificationType;
import notification.service.model.Person;

public record NotificationRequest(String message,
                                  Person person,
                                  @JsonProperty("notificationType") NotificationType type) {
}

File: ticket-service/src/main/kotlin/ru/mcr/ticketservice/dto/NotificationRequest.kt
package ru.mcr.ticketservice.dto

import lombok.AllArgsConstructor
import lombok.Builder
import lombok.Data
import lombok.NoArgsConstructor
import ru.mcr.ticketservice.NotificationType

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
class NotificationRequest(
    var message: String? = null,
    var person: Person? = null,
    var notificationType: NotificationType? = null,
)